jwst/pipeline/outlier_detection_stack.cfg

```diff
@@ -1,17 +1,4 @@
 name = "outlier_detection_stack" 
 class = "jwst.outlier_detection_stack.OutlierDetectionStackStep"
 
-weight_type = 'exptime'
-pixfrac = 1.0
-kernel = 'square'
-fillval = 'INDEF'
-nlow = 0
-nhigh = 0
-maskpt = 0.7
-grow = 1
-snr = '4.0 3.0'
-scale = '0.5 0.4'
-backg = 0.0        
-save_intermediate_results = False
 resample_data = False
```

Comment:
`resample_data = False` in the step spec, so you can probably remove it here too.

Reply:
Ah, right. The default is `True` in the regular `OutlierDetectionStep`, so I thought I'd leave it in. But I see that the defaults are different in `OutlierDetectionStackStep`.

Reply:
Done